File: Backend/FitTrackWithMLP/FitTrackWithMLP.Shared/DTOs/UserPhysiqueDto.cs
namespace FitTrackWithMLP.Shared.DTOs
{
    public class UserPhysiqueDto
    {
        public int Tdee { get; set; }
        public int Weight { get; set; }
        public string ActivityLevel { get; set; } = string.Empty;
        public string GoalType { get; set; } = string.Empty;
        public string MealsComplexity { get; set; } = "Standard";
    }
}

File: Frontend/src/dtos/UserPhysiqueDto.ts
class UserPhysiqueDto {
    tdee: number;
    weight: number;
    activityLevel: string;
    goalType: string;
    mealsComplexity: string;

    public constructor(tdee: number, weight: number, activityLevel: string, goalType: string, mealsComplexity: string = "Standard") {
        this.tdee = tdee;
        this.weight = weight;
        this.activityLevel = activityLevel;
        this.goalType = goalType;
        this.mealsComplexity = mealsComplexity;
    }
}

export default UserPhysiqueDto;